crates/chain/src/tx_graph.rs

```diff
@@ -449,6 +449,32 @@ impl<A> TxGraph<A> {
     pub fn is_empty(&self) -> bool {
         self.txs.is_empty()
     }
+
+    /// Map TxGraph for another anchor type
```

Comment:
```suggestion
    /// Transform the [`TxGraph`] to have [`Anchor`]s of another type.
    ///
    /// This takes in a closure of signature `Fn(A) -> A2` which is called for each [`Anchor`] to
    /// transform it.
```


Some notes on writing documentation:

* Use the following structure (as suggested [here](https://doc.rust-lang.org/rustdoc/how-to-write-documentation.html#documenting-components)), but maybe skip the example for simple methods (such as this one):

    ```
    [short sentence explaining what it is]
    
    [more detailed explanation]
    
    [at least one code example that users can copy/paste to try it]
    
    [even more advanced explanations if necessary]
    ```

* Use links when referencing structures (such as `TxGraph`).